src/controller/login.js

```diff
@@ -1,51 +1,119 @@
-import Layout from "component/layout";
+import ModelAccount from "model/account";
+import ModelUser from "model/user";
+import AppRequest from "component/request";
 
-var LoginController = {
+import LoginView from "view/login";
+
+import LocalForage from "localforage";
+import Storage from "model/storage";
+
+export default {
     controller: function() {
-        var storage = window.localStorage;
-        return {
-            storage: storage,
-            data: {
-                token: m.prop(LoginController.getToken(storage)),
-                endpoint: m.prop(LoginController.getEndpoint(storage))
+
+        this.accounts = m.prop([]);
+        this.users = m.prop([]);
```

Comment:
> Note that modifying the values of m.prop getter-setters does not trigger redrawing.

From: https://lhorie.github.io/mithril/mithril.prop.html

Redrawing is triggered when a component is initialized or when an event is handled, along with `m.prop` being updated. You can also hint for redrawing by using `m.start/endComputation`.